client/src/components/pages/application/Project/style.css

```diff
@@ -0,0 +1,83 @@
+
+.Container_page {
+  background-color: rgb(245, 245, 245);
+  width: 100vw;
+  height: 100vh;
+}
```

Comment:
There is horizontal scroll in your design and this is not acceptable, please try to fix it .